src/Spout/Reader/CSV/Reader.php

```diff
@@ -32,6 +32,9 @@ class Reader extends AbstractReader
     /** @var string Defines the End of line */
     protected $endOfLineCharacter = "\n";
 
+    /** @var bool */
```

Comment:
my bad, this should be a string. ini_get/ini_set returns/takes a string as argument